modules/encoders/x86/unicode_upper.rb

```diff
@@ -12,7 +12,7 @@ def initialize
     super(
       'Name'             => "Alpha2 Alphanumeric Unicode Uppercase Encoder",
       'Description'      => %q{
-        Encodes payload as unicode-safe uppercase text.  This encoder uses
+        Encodes payloads as unicode-safe uppercase text.  This encoder uses
```

Comment:
I don't think this is a necessary change. You can encode only one payload at a time. It is specific language.

Reply:
I was trying to remove it from the unicode_mixed and ended up adding an s to the unicode_upper, good catch.